app/services/hot_lunch/sum_child_cost.rb

```diff
@@ -1,5 +1,10 @@
 class HotLunch::SumChildCost < ChargesService
   REQUIRED_AGES = 3..4
+  IS_MULTI_WEEK_CONF = false
+  # System was built for a conference over 4 weeks, so charges were done per week.  
+  # Now there is only a single week, but may go back, so left code and a flag {#IS_MULTI_WEEK_CONF}
+  # Currently using the 5th entry in hot lunch weeks (0 index so 4) 
+  WEEK_INDEX = 4
```

Comment:
Factor these out into the superclass?

Reply:
Not sure if its worth it right now, it was implemented in a temporary way while Andy determines if they want to go with the Multiweek conference system or not in the future or the ability to switch.  Once he decides then we can adjust this either by removing entirely or factoring it out.  However, this project was setup on a last minute project and tight deadline for the conference (Tuesday this week) so certain features like this will be updated/added after launch of this functionality